app/src/main/java/org/oppia/app/activity/HtmlParserTestActivty.kt

```diff
@@ -1,22 +1,26 @@
 package org.oppia.app.activity
```

Comment:
I would suggest to shift this HtmlParserTestActivity to `testing` package or some other package with implements `HtmlParser`

Reply:
moved it to package testing.